Switch/Switch.js

```diff
@@ -25,16 +25,16 @@ import componentCss from './Switch.module.less';
 /**
  * Renders the base level DOM structure of the component.
  *
- * @class Switch
+ * @class SwitchBase
  * @memberof agate/Switch
- * @extends agate/ToggleIcon.ToggleIcon
+ * @extends ui/ToggleIcon.ToggleIconBase
  * @ui
  * @public
  */
 const SwitchBase = kind({
 	name: 'Switch',
 
-	propTypes: /** @lends agate/Switch.Switch.prototype */ {
+	propTypes: /** @lends agate/Switch.SwitchBase.prototype */ {
```

Comment:
In some of the other changes, you went from `agate/Component.ComponentBase.prototype` to `agate/Component.Component.prototype`, is there a reason for them to differ?

Reply:
Yes, in those cases there is no (documented) `ComponentBase`.

Reply:
Ah, got it.